plugin/ensime.vim

```diff
@@ -38,6 +37,10 @@ command! -nargs=* -range EnToggleFullType call ensime#com_en_toggle_fulltype([<f
 command! -nargs=* -range EnOrganizeImports call ensime#com_en_organize_imports([<f-args>], '')
 command! -nargs=* -range EnAddImport call ensime#com_en_add_import([<f-args>], '')
 
+function! EnPackageDecl() abort
+  return ensime#com_en_package_decl()
+endfunction
```

Comment:
> Are you up for a quick PR to remove those methods and make those renamings? Since you're the one that proposed them, I think it's fair if you carry them out.

Sure, it seems that it's been on my mind already so I ought to act on it 😇  Just trying to find the time… The renaming part should be quick but other aspects might lead deeper. I'll start with #243.